internal/store/device.go

```diff
@@ -36,6 +36,7 @@ type Device interface {
 	UpdateRendered(ctx context.Context, orgId uuid.UUID, name, renderedConfig, renderedApplications string) error
 	GetRendered(ctx context.Context, orgId uuid.UUID, name string, knownRenderedVersion *string, consoleGrpcEndpoint string) (*api.RenderedDeviceSpec, error)
 	SetServiceConditions(ctx context.Context, orgId uuid.UUID, name string, conditions []api.Condition) error
+	SetDecommission(ctx context.Context, orgId uuid.UUID, name string, request *api.DeviceDecommission) error
```

Comment:
I'm currently standardizing the store API across resources and reducing the number of methods.  Can we use Update() instead, perhaps by passing fromAPI=false and thus avoid the checks that you added there?

Reply:
Okay, let me take a look at doing it that way.

Reply:
Updated - no pun intended :)